README.md

```diff
@@ -1,24 +1,14 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Thanks for taking a look at this!
 
-Things you may want to cover:
+A couple of questions:
```

Comment:
Thanks for specifying what you want review on! 👏🏽 I think in the future, it would be better to add these comments to the description of the Pull Request, that way you can easily edit/update it with any new questions and this files changed tab can be reserved for actual code changes.